pkg/skaffold/jib/jib_darwin.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package jib
+
+func getWrapper() string {
+	return "gradlew"
+}
+
+func getCommand(workspace string, defaultExecutable string, defaultSubCommand []string) (executable string, subCommand []string) {
+	executable = defaultExecutable
+	subCommand = defaultSubCommand
+
+	wrapperExecutable := getWrapper()
+	if exists(workspace, wrapperExecutable) {
+		executable = "./" + wrapperExecutable
```

Comment:
I think it's worth making the path absolute just in case the command-execution sets a different working directory